opensrp-giz-malawi/src/main/java/org/smartregister/giz/listener/OnLocationChangeListener.java

```diff
@@ -0,0 +1,5 @@
+package org.smartregister.giz.listener;
+
+public interface OnLocationChangeListener {
+    void updateUi(String location);
```

Comment:
Annotate this method to indicate that we can handle nulls, Also do the same to the implementations